File: found/calc/count/count..py
# counts the number of sheep present in the array (true means present).

def count_sheeps(sheep_array):
    count = 0
    for sheep in sheep_array:
        if sheep is True:
            count += 1
    return count

def count_sheeps(sheep_array):
    return sum(1 for sheep in sheep_array if sheep is True)

# sheep_array = [None, None, False, True, True, False, None, None]
# print(count_sheeps(sheep_array))

File: found/calc/count/count.js
// counts the number of sheep present in the array (true means present).

function countSheeps(sheepArray) {
  let count = 0;

  for (let i = 0; i < sheepArray.length; i++) {
    if (sheepArray[i] === true) {
      count++;
    }
  }
  return count;
}
// [undefined,null,false,true,true,false,null,undefined]
